pytorch_lightning/plugins/training_type/deepspeed.py

```diff
@@ -562,6 +562,11 @@ def _format_batch_size_and_grad_accum_config(self):
                 " as this will be set via accumulate_grad_batches=x argument passed via the Lightning Trainer."
             )
         if "train_micro_batch_size_per_gpu" not in self.config:
+            rank_zero_warn(
+                "Inferring the batch size for internal deepspeed logging from the ``train_dataloader()``. "
+                "If you require skipping this, please pass "
+                "``Trainer(plugins=DeepSpeedPlugin(logging_batch_size_per_gpu=X)``, where X is the batch size."
```

Comment:
double backticks are for rendered docs

```suggestion
                "Inferring the batch size for internal deepspeed logging from the `train_dataloader()`. "
                "If you require skipping this, please pass "
                "`Trainer(plugins=DeepSpeedPlugin(logging_batch_size_per_gpu=batch_size)`"
```